From 86ce7047b2ade1f18ed50a736f10c343483f1f6c Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Thu, 9 Feb 2023 13:55:47 +0100
Subject: [PATCH] fix(systemd-pcrphase): only include
 systemd-pcrphase-initrd.service

The only systemd-pcrphase related unit configured to run in the initrd is
systemd-pcrphase-initrd.service.
Both systemd-pcrphase.service and systemd-pcrphase-sysinit.service contain
`ConditionPathExists=!/etc/initrd-release`.

(Cherry-picked commit: cd6f683d634970112a29867137431d0d57f8c957)

Related: RHEL-27944
---
 modules.d/01systemd-pcrphase/module-setup.sh | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/modules.d/01systemd-pcrphase/module-setup.sh b/modules.d/01systemd-pcrphase/module-setup.sh
index 3dbb4974..fa960a42 100755
--- a/modules.d/01systemd-pcrphase/module-setup.sh
+++ b/modules.d/01systemd-pcrphase/module-setup.sh
@@ -28,10 +28,6 @@ install() {
 
     inst_multiple -o \
         "$systemdutildir"/systemd-pcrphase \
-        "$systemdsystemunitdir"/systemd-pcrphase.service \
-        "$systemdsystemunitdir/systemd-pcrphase.service.d/*.conf" \
-        "$systemdsystemunitdir"/systemd-pcrphase-sysinit.service \
-        "$systemdsystemunitdir/systemd-pcrphase-sysinit.service/*.conf" \
         "$systemdsystemunitdir"/systemd-pcrphase-initrd.service \
         "$systemdsystemunitdir/systemd-pcrphase-initrd.service.d/*.conf" \
         "$systemdsystemunitdir"/initrd.target.wants/systemd-pcrphase-initrd.service
@@ -39,10 +35,6 @@ install() {
     # Install the hosts local user configurations if enabled.
     if [[ $hostonly ]]; then
         inst_multiple -H -o \
-            "$systemdsystemconfdir"/systemd-pcrphase.service \
-            "$systemdsystemconfdir/systemd-pcrphase.service.d/*.conf" \
-            "$systemdsystemconfdir"/systemd-pcrphase-sysinit.service \
-            "$systemdsystemconfdir/systemd-pcrphase-sysinit.service.d/*.conf" \
             "$systemdsystemconfdir"/systemd-pcrphase-initrd.service \
             "$systemdsystemconfdir/systemd-pcrphase-initrd.service.d/*.conf" \
             "$systemdsystemconfdir"/initrd.target.wants/systemd-pcrphase-initrd.service

